doc/Language/math.pod6

```diff
@@ -228,11 +228,255 @@ The L<Math::Constants|https://github.com/JJ/p6-math-constants/pulls> module
+Let's start with the I<'Hello World'> of mathematical Ecology: 
+L<Malthusian growth model|https://en.wikipedia.org/wiki/Malthusian_growth_model>. 
+A Malthusian growth model, sometimes called a simple exponential growth
+ model, is essentially exponential growth based on the idea of the 
+ function being proportional to the speed to which the function grows.
+The equation, then, looks like this: 
+
+I<dx/dt = g*x>
+
+I<x(0) = x_0> 
+
+Where I<g> is the population growth rate, sometimes called Malthusian 
+parameter.
+
+How can we translate that into Perl6? Well Math::Model bring some help
+with a very undestable way to do that, let's see it:
+
+=begin code
+use Math::Model;
+
+my $m = Math::Model.new(
+    derivatives => {
+        velocity     => 'x',
+    },
+    variables   => {
+        velocity           => { $:growth_constant * $:x},
+        growth_constant    => { 1 },   # basal growth rate
+    },
+    initials    => {
+        x       => 3,
+    },
+    captures    => ('x'),
+);
+
+$m.integrate(:from(0), :to(8), :min-resolution(0.5));
+$m.render-svg('population growth malthus.svg', :title('population growth'));
+=end code
+
+To fully understand what is going on, let me explain it step by step.
+
+=head3 Step by step explanation.
+
+=begin item 
+First we load the module that make 
+the calculations: L<C<Math::Model>|https://github.com/moritz/Math-Model>.
+
+=begin code :skip-test<Snippet for explanantion purposes>
+use Math::Model;
+=end code
+=end item 
+
+=begin item 
+We create the model to add all the information in it.
+=begin code :skip-test<Snippet for explanantion purposes>
+my $m = Math::Model.new(
+=end code
+=end item 
+
+=begin item 
+We declare the derivatives that are in our model. In this case, if
+you remember our equation, we have our variable I<x> and its derivative 
+I<x'> (usually know as the velocity).
+
+=begin code :skip-test<Snippet for explanantion purposes>
+derivatives => {
+    velocity     => 'x',
+    },
+=end code
+=end item 
+
+=begin item 
+After that, we declare how our models evolve. We just need 
+formulas for the derivatives that are not also integration variables 
+(in this case, only I<x>), and for other variables we use in the formulas
+ (the growth rate).
+
+=begin code :skip-test<Snippet for explanantion purposes>
+variables   => {
+    velocity           => { $:growth_constant * $:x},
+    growth_constant    => { 1 },   # basal growth rate
+},
+=end code
+=end item 
+
+
+=begin item 
+Finally we declare our initial conditions and use I<captures> to 
+tell L<C<Math::Model>|https://github.com/moritz/Math-Model> which 
+variable or variables to record while the simulation is running.
+
+=begin code :skip-test<Snippet for explanantion purposes>
+initials    => {
+    x       => 3,
+},
+captures    => ('x'),
+=end code
+=end item 
+
+
+At this point our model is set. We need to run the simulation and render
+ a cool plot about our results:
+=begin code :skip-test<Snippet for explanantion purposes>
```

Comment:
I would say this snippet compiles, no need to skip. If the examples are correct Perl 6 sentences, you can leave them like that.